.github/workflows/release-workflow.yml

```diff
@@ -85,8 +85,9 @@ jobs:
           file_pattern: 'documentation/antora.yml pom.xml */pom.xml */*/pom.xml'
 
       - name: Build and Deploy to Maven Central
-        run: mvn -B clean deploy -Psign
+        run: mvn -B clean deploy org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Psign
```

Comment:
By looking at your build output, I recommend to keep the Sonar cache and reuse it

Reply:
Do you mean sonar cache or maven cache?

Reply:
Saw that you added the Maven caching into this PR, that's great.  I was referring to Sonar in the context of this PR:
```
- name: Cache SonarQube packages
    uses: actions/cache@v4
    with:
      path: ~/.sonar/cache
      key: ${{ runner.os }}-sonar
      restore-keys: ${{ runner.os }}-sonar
```

Reply:
@schabdo thanks added it to both pipelines